src/shared/components/booth-detail-item/booth-detail-item.tsx

```diff
@@ -0,0 +1,44 @@
+import * as style from './booth-detail-item.css';
+import Thumbnail from '../Thumbnail/Thumbnail';
+
+interface BoothDetailItemProps {
+  itemName: string;
+  itemInfo: string;
+  itemImg: string;
+  itemAlt: string;
```

Comment:
어차피 item에 대한 내용이니 props의 이름도
```suggestion
  name: string;
  info: string;
  imgSrc: string;
  imgAlt: string;
```
이런식으로 해도 될것 같아요! 그리고 Thumbnail에서 src에 대한 내용이니 imgSrc가 괜찮지 않을까 생각이 듭니다!!